.circleci/config.yml

```diff
@@ -101,6 +101,14 @@ workflows:
           filters:
             tags:
               only: /^v([0-9])+.([0-9])+.([0-9])+.*/
+      - windows-msi:
+          requires:
+            - cross-compile
+          filters:
+            branches:
+              only: master
```

Comment:
@james-bebbington I think we should remove the branches filter and run for all PRs.  Otherwise, the job won't get executed for PRs (like this one) and we wouldn't have any visibility into it until after it gets merged to master.